tiddlers/stylesheet.tid

```diff
@@ -1,64 +1,135 @@
 title: $:/plugins/tgrosinger/tw5-checklist/stylesheet
 tags: $:/tags/Stylesheet
-type: text/css
 
 ul.checklist {
     list-style: none;
     margin: 0;
-    padding: 0;
+    padding: 0 0 0 1.5em;
 }
 
 ul.checklist > li {
     position: relative;
+    width: calc(100% - 1.5em);
 }
 
-ul.checklist > li:first-child {
-    margin-bottom: 2px;
-}
 
-span.checklistitem-body {
-    margin-left: 5px;
-}
+/* input form */
 
-span.checkedchecklistitem-body {
-    color: #ddd;
-    text-decoration: line-through;
+.checklist-newitem-icon::before {
+    content: "&#x270e;";
 }
 
 input.checklist-newitem {
-    margin-left: 5px;
+    margin-left: 1em;
+    margin-bottom: 0.75em;
     padding: 2px;
     border: none;
-    border-bottom: solid 2px #c9c9c9;
+    border-bottom: solid 2px <<colour sidebar-tab-background>>;
     transition: border 0.3s;
+    width: calc(100% - 2em - 20px);
 }
 
 input:focus.checklist-newitem {
-    border-bottom: solid 2px #969696;
+    border-bottom: solid 2px <<colour notification-border>>;
+}
+
+
+/* placeholder text */
+
+::-webkit-input-placeholder { 
+  color: #444;
+  font-style: italic;
+}
+::-moz-placeholder { 
+  color: #444;
+  font-style: italic;
+}
+:-ms-input-placeholder { 
+  color: #444;
+  font-style: italic;
+}
+:-moz-placeholder { 
+  color: #444;
+  font-style: italic;
+}
+
+
+/* list items content */
+
+label.checklistitem-body {
+    margin-left: -0.9em;
+    margin-right: 3em;
+    position: relative;
+    left: 1.75em;
+}
+
+input:checked + label.checklistitem-body {
+    color: <<colour muted-foreground>>;
 }
 
-div.checklist-removeitem-icon {
-    bottom: 0;
-    color: #777;
+<$list filter="[[$:/plugins/tgrosinger/tw5-checklist/Configuration]field:strike-checked[true]]">
+input:checked + label.checklistitem-body {
+    text-decoration: line-through;
+}
+</$list>
+
+
+/* buttons */
+
+button.checklist-add {
+    position: relative;
+    left: 0.75em;
+    width: 18px;
+    height: 18px;
+    border-radius: 5px;
+background-image: url('data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%2211%22%20height%3D%2211%22%3E%0A%3Crect%20width%3D%221%22%20height%3D%227%22%20x%3D%225%22%20y%3D%222%22%20fill%3D%22%239B9B9B%22%2F%3E%0A%3Crect%20width%3D%227%22%20height%3D%221%22%20x%3D%222%22%20y%3D%225%22%20fill%3D%22%239B9B9B%22%2F%3E%0A%3C%2Fsvg%3E');
```

Comment:
Is this just a small "+" icon? Can we just use some sort of unicode character instead?